### PR TITLE
sku-level-backend-1.0-issue #26

### PR DESCRIPTION
**Pull request by strikeyamato**
Added utils.py for check_sku_alerts function which has a filter. Created a model, and updated the views.py and urls.py files. Ensured sku_color is the primary key based on recent db changes.

TO DO: Create automatic trigger for the alerts

### CODE REVIEW

**Comment by Irisvella** [comment]
There are conflicts to resolve

**Comment by strikeyamato** [comment]
Conflicts resolved. Awaiting review

File: WarehousePilot_app/backend/manager_dashboard/urls.py
Comment (Irisvella) [inline WarehousePilot_app/backend/manager_dashboard/urls.py:1]: remove profileview and changepassword view. They dont go there

File: WarehousePilot_app/backend/manager_dashboard/views.py
Comment (Irisvella) [inline WarehousePilot_app/backend/manager_dashboard/views.py:1]: same here, the APIs dont go here

**Review by Irisvella — CHANGES_REQUESTED** [review]

**Review by strikeyamato — COMMENTED** [review]
removed profileview and changepassword from views.py and urls.py

**Review by Irisvella — APPROVED** [review]

File: WarehousePilot_app/backend/manager_dashboard/urls.py
Comment (strikeyamato) [reply WarehousePilot_app/backend/manager_dashboard/urls.py:1]: Removed

**Comment by yousefenein** [comment]
looks good, just need to integrate this with inventory

**Comment by yousefenein** [comment]
looks good, just need to integrate this with inventory

**Review by yousefenein — APPROVED** [review]
looks good!

**Pull request merged**